index.bs

```diff
@@ -3296,6 +3368,92 @@ failed</dfn> steps given |context| and |navigation status|:
 
 </div>
 
+#### The browsingContext.userPromptClosed Event #### {#event-browsingContext-userPromptClosed}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        BrowsingContextUserPromptClosedEvent = {
+         method: "browsingContext.userPromptClosed",
+         params: UserPromptInfo
+       }
+       UserPromptResult = {
+         context: BrowsingContext,
+         confirmed: bool,
```

Comment:
Should we use the same terminology as in the command, `accepted`? Or is there a difference?